src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -339,7 +339,7 @@ public function present($cart, $shouldSeparateGifts = false)
         if ($total_discount) {
             $subtotals['discounts'] = array(
                 'type' => 'discount',
-                'label' => $this->translator->trans('Discount', array(), 'Shop.Theme.Checkout'),
+                'label' => $this->translator->trans('Discount(s)', array(), 'Shop.Theme.Checkout'),
```

Comment:
@LouiseBonnard don't you think it should always be in plural?

Reply:
As it can be in a singular form as well, I prefer allowing both with this wording.